Outlands_Webminister_Handbook.md

```diff
@@ -20,12 +20,18 @@ The kingdom website shall maintain a library of current governing documents and
 
 ### 2.2 Other Internet Infrastructure
 
+The Kingdom Webminister should provide official outlands.org email accounts for kingdom officers. All official correspondence should come from the provided email.
```

Comment:
I consider this an open topic, specifically whether the "web minister" office is actually "kingdom IT" or if that would be better placed under the Seneschal.  Haven't started the discussion yet.